frontend/src/services/project.ts

```diff
@@ -48,3 +48,68 @@ export async function createProject(projectData: ProjectForm): Promise<number> {
     });
     return response.id;
 }
+
+export async function updateProject(
+    projectId: number,
+    projectData: Partial<ProjectForm>
+): Promise<void> {
+    const response = await authorized_fetch(`/api/projects/${projectId}`, {
+        method: "PATCH",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(projectData),
+    });
+    return;
+}
+
+// Function to upload test files to a specific project
+export async function uploadProjectFiles(projectId: number, formData: FormData): Promise<void> {
+    await authorized_fetch(
+        `/api/projects/${projectId}/test_files`,
+        {
+            method: "PUT",
+            body: formData,
+        },
+        { omitContentType: true }
+    );
+}
+
+export async function fetchProjectFiles(projectId: number): Promise<any> {
+    return authorized_fetch(`/api/projects/${projectId}/test_files`, {
+        method: "GET",
+    })
+        .then((response) => {
+            return response;
+        })
+        .catch((error) => console.error("Failed to fetch project files:", error));
+}
```

Comment:
Het is niet de bedoeling om hier errors op te vangen.

Reply:
done